src/jit/compiler.h

```diff
@@ -1918,7 +1925,7 @@ class fgArgInfo
     // Caller must ensure that this index is a valid arg index.
     GenTree* GetArgNode(unsigned argIndex)
     {
-        return GetArgEntry(argIndex)->node;
+        return GetArgEntry(argIndex)->GetNode();
```

Comment:
a random thing:
in `GenTreeCall::Use::SetNode ` we check that `assert(node != nullptr);`, so as I understand our contract is that if we have a `Use` then we have a not-null node, but we don't check that in the Use constructor, so we can call `GenTreeCall::Use(nullptr)` and do bad things.

Reply:
Right, I need to check this. AFAIR I didn't add an assert in the `Use` constructor due to some funny code in `fgArgInfo::fgArgInfo` that was manufacturing uses out of thin air. That code is gone now so perhaps I can add an assert.